ci/clippy.sh

```diff
@@ -6,5 +6,5 @@
 #
 # https://users.rust-lang.org/t/pre-commit-clippy-fix/66584
 
-cargo clippy --all-targets --workspace --fix --allow-dirty --allow-staged
+cargo clippy --all-targets --workspace --fix --allow-dirty --allow-staged --allow-no-vcs
```

Comment:
I hit this too on my branch and cherry-picked this commit. It's weird that this is happening (I would have thought the Github Actions environment has a bona fide Git checkout).